action.yml

```diff
@@ -1,23 +1,72 @@
-name: 'Example composite GitHub action'
-description: 'Example composite GitHub action'
+name: 'GitHub Action Atmos Component Updater'
+description: 'GitHub Action Atmos Component Updater'
 author: hello@cloudposse.com
 branding:
-  icon: 'file'
+  icon: 'server'
   color: 'white'
 inputs:
-  param1:
+  token:
     required: true
-    description: "Input parameter placeholder"
-    default: "true"
-outputs:
-  result1:
-    description: "Output result placeholder"
-    value: "${{ steps.context.outputs.action-result }}"
+    description: "GitHub Private Access Token"
+  desired-components-version:
+    required: false
+    description: "Desired Components Version. If not set it gets latest tag from cloudposse/terraform-aws-components repo"
```

Comment:
`cloudposse/terraform-aws-components` should be parameterizable since end-users might have their own component repo or even pull from multiple repos.

Reply:
Also, I'm not sure how useful this is. I could imagine using semver to pin individual components, but having one hardcoded version for everything seems odd.